firetower

```diff
@@ -216,8 +216,6 @@ function get_directory() {
   echo "./"
 }
 
-
-
```

Comment:
for a small amend commits like this, make sure the commit message is accurate, since this doesn't actually fix the flags, a good commit message might be something like `remove accidentally added newlines`

especially with github, if a commit is merged with a message that contains something like `fixes #7`, then  ticket #7 will be closed (see https://help.github.com/articles/closing-issues-via-commit-messages/)